Libraries/LibGfx/Rect.h

```diff
@@ -99,6 +99,20 @@ class Rect {
         m_location.move_by(delta);
     }
 
+    Rect moved_by(T dx, T dy) const
```

Comment:
This already exists; it's called `translated()` :)

Reply:
Ah. weird naming conflict, but ok.